.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -7,29 +7,40 @@ assignees: ''
 
 ---
 
-### Description 
+### Description
 
-<!-- role details / capability details / value details -->
-As <role> I want <capability> so I can <value>
+<!-- For you!
+Please provide a concise description of what this story is about.
+The story should contain the following information:
+    * Who is going to need this? What role can you identify? Try to be as specific as possible. It helps us to understand the need.
+    * What would you like to have? Add as much information as possible so we can understand the problem at any time.
+    * How does it benefit your work? This helps us to understand the value of the story and how to prioritise the work.
 
+To help you formulating your story you can use one of the following phrases:
+    * As <role> I want <capability> so I can <value>
+    * <action> the <result> <by|for|of|to> a(n) <object>
+-->
 
 ### Acceptance Criteria
 
-<!-- All the stuff that needs to be done, besides the Definition of Done for this story. -->
+<!-- For Triage!
+This section is filled later when the story is in triage:
+    * Ask all relevant questions to understand the functionality, its dependencies and implications.
+    * Estimate the work and split if necessary.
+    * Define specifications in form of testable acceptance criteria.
 
-- [ ] test 1
-- [ ] test 2
-- [ ] test 3
+In triage phase it is important to ask all relevant questions about the story such that everyone understands the functionality of the story, as well as its implications and dependencies. The story will then be estimated and possibly split into smaller components. Further, the triage team has to write down the story specifications in form of acceptance criteria that need to be tested. After completing the triage phase the issue will be handed to the dev team.
+ -->
 
 ### Tasks
 
-<!-- The tasks which need to be tackled for this story to be completed. They are likely to be defined later when the team plans this story for the next iteration and discusses the design. --> 
+<!-- For Development!
+This section is filled by the development team to specify the concrete tasks to solve the story.
```

Comment:
Who are these teams? triage team,  development team? 
Is one of them the Core team?